src/utils/fishData.js

```diff
@@ -33,7 +41,7 @@ const fishComponents = {
       pectoralFinFrontAnchor: [55, 72],
       dorsalFinAnchor: [23, -15],
       tailAnchor: [107, 41],
-      knnData: [1.46, 7120],
+      knnData: [7120, BodyShape.OVAL],
```

Comment:
What do these numbers mean, e.g. 7120?

Reply:
Comment on line 33 explains. I forgot to update and am pushing that updated comment now.